api/mixins.py

```
@@ -0,0 +1,51 @@
+from rest_framework.exceptions import ValidationError
+
+
+class SerializerMixin:
+    serializer_class: typing.Optional[typing.Type[Serializer]] = None
+
+    def get_serializer_class(self) -> typing.Type[Serializer]:
+        if self.serializer_class is None:
+            raise ValidationError("serializer_class must not be None")
+        return self.serializer_class
+
+    def deserialize(self, data={}) -> Serializer:
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)